packages/fxa-shared/test/metrics/amplitude.js

```diff
@@ -138,6 +140,7 @@ describe('metrics/amplitude:', () => {
 
       it('returned the correct result', () => {
         assert.deepEqual(result, {
+          app_version: '0',
```

Comment:
Is this new?

Reply:
Side effect of the monorepo. Previously, the `../../../package.json` in `metrics/amplitude.js` would only resolve to something when run from `node_modules`, so app_version would not be set when the tests were run locally and in CI.

But now it resolves to the top-level `package.json`, which has `0` in the train number position.